json/schema/user_import.schema.json

```diff
@@ -51,15 +51,11 @@
           },
           "country_code": {
             "description": "User's country, 2 letter country code, uppercase, optional.",
-            "type": "string",
-            "pattern": "^[A-Z]{2}$",
-            "minLength": 1
+            "type": "string"
           },
           "locale": {
             "description": "User's locale string in ISO/IEC 15897 format, e.g., en_GB, pt_BR, etc. Optional.",
-            "type": "string",
-            "pattern": "^[a-z]{2}_[A-Z]{2}$",
```

Comment:
Why pattern got removed as well?

Reply:
pattern didn't match as well for empty string, and both fields have validators now

Reply:
If empty string is a valid value please use `"anyOf": [ ]` and keep the pattern.